ee/app/livechat-enterprise/server/hooks/afterRemoveDepartment.js

```diff
@@ -0,0 +1,9 @@
+import { callbacks } from '../../../../../app/callbacks';
+import { LivechatDepartment } from '../../../../../app/models/server';
+
+callbacks.add('livechat.afterRemoveDepartment', (department) => {
+	if (!department) {
+		return;
+	}
+	LivechatDepartment.removeDepartmentFromForwardListById(department._id);
```

Comment:
Same here. Since we can have several callbacks for the same event and they are executed in a sequenced way, the next callback we need to get the original parameter.

```suggestion
	LivechatDepartment.removeDepartmentFromForwardListById(department._id);
	return department;
```